src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -337,4 +400,50 @@ protected function getGridActions()
                     ->setIcon('storage')
             );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getBulkActions()
+    {
+        return (new BulkActionCollection())
+            ->add(
+                (new AjaxBulkAction('enable_selection_ajax'))
+                    ->setName($this->trans('Activate selection', [], 'Admin.Actions'))
+                    ->setOptions([
+                        'ajax_route' => 'admin_products_v2_activate_ajax',
+                        'modal_title' => $this->trans('Activating %d products', [], 'Admin.Actions'),
```

Comment:
I had made some comments about this part. It appears "outdated" even though the changes weren't applied:
![image](https://user-images.githubusercontent.com/70583503/162218160-636528e4-80af-40b5-99d8-5e5e6a421293.png)